recipes-hulks/aliveness/aliveness_git.bb

```diff
@@ -0,0 +1,190 @@
+    crate://crates.io/regex/1.7.0 \
+    crate://crates.io/remove_dir_all/0.5.3 \
+    crate://crates.io/rustc-demangle/0.1.21 \
+    crate://crates.io/rustix/0.36.7 \
+    crate://crates.io/ryu/1.0.11 \
+    crate://crates.io/scopeguard/1.1.0 \
+    crate://crates.io/serde/1.0.149 \
+    crate://crates.io/serde_derive/1.0.149 \
+    crate://crates.io/serde_json/1.0.92 \
+    crate://crates.io/serde_repr/0.1.10 \
+    crate://crates.io/sha1/0.10.5 \
+    crate://crates.io/sharded-slab/0.1.4 \
+    crate://crates.io/signal-hook-registry/1.4.0 \
+    crate://crates.io/slab/0.4.7 \
+    crate://crates.io/smallvec/1.10.0 \
+    crate://crates.io/socket2/0.4.7 \
+    crate://crates.io/static_assertions/1.1.0 \
+    crate://crates.io/syn/1.0.105 \
+    crate://crates.io/tempfile/3.3.0 \
+    crate://crates.io/termcolor/1.2.0 \
+    crate://crates.io/thiserror-impl/1.0.37 \
+    crate://crates.io/thiserror/1.0.37 \
+    crate://crates.io/thread_local/1.1.4 \
+    crate://crates.io/tokio-macros/1.8.2 \
+    crate://crates.io/tokio-util/0.7.4 \
+    crate://crates.io/tokio/1.23.0 \
+    crate://crates.io/toml/0.5.10 \
+    crate://crates.io/tracing-attributes/0.1.23 \
+    crate://crates.io/tracing-core/0.1.30 \
+    crate://crates.io/tracing-error/0.2.0 \
+    crate://crates.io/tracing-subscriber/0.3.16 \
+    crate://crates.io/tracing/0.1.37 \
+    crate://crates.io/typenum/1.16.0 \
+    crate://crates.io/uds_windows/1.0.2 \
+    crate://crates.io/unicode-ident/1.0.5 \
+    crate://crates.io/valuable/0.1.0 \
+    crate://crates.io/value-bag/1.0.0-alpha.9 \
+    crate://crates.io/version_check/0.9.4 \
+    crate://crates.io/waker-fn/1.1.0 \
+    crate://crates.io/wasi/0.11.0+wasi-snapshot-preview1 \
+    crate://crates.io/wasm-bindgen-backend/0.2.83 \
+    crate://crates.io/wasm-bindgen-futures/0.4.33 \
+    crate://crates.io/wasm-bindgen-macro-support/0.2.83 \
+    crate://crates.io/wasm-bindgen-macro/0.2.83 \
+    crate://crates.io/wasm-bindgen-shared/0.2.83 \
+    crate://crates.io/wasm-bindgen/0.2.83 \
+    crate://crates.io/web-sys/0.3.60 \
+    crate://crates.io/wepoll-ffi/0.1.2 \
+    crate://crates.io/winapi-i686-pc-windows-gnu/0.4.0 \
+    crate://crates.io/winapi-util/0.1.5 \
+    crate://crates.io/winapi-x86_64-pc-windows-gnu/0.4.0 \
+    crate://crates.io/winapi/0.3.9 \
+    crate://crates.io/windows-sys/0.42.0 \
+    crate://crates.io/windows_aarch64_gnullvm/0.42.0 \
+    crate://crates.io/windows_aarch64_msvc/0.42.0 \
+    crate://crates.io/windows_i686_gnu/0.42.0 \
+    crate://crates.io/windows_i686_msvc/0.42.0 \
+    crate://crates.io/windows_x86_64_gnu/0.42.0 \
+    crate://crates.io/windows_x86_64_gnullvm/0.42.0 \
+    crate://crates.io/windows_x86_64_msvc/0.42.0 \
+    crate://crates.io/zbus/3.7.0 \
+    crate://crates.io/zbus_macros/3.7.0 \
+    crate://crates.io/zbus_names/2.5.0 \
+    crate://crates.io/zvariant/3.10.0 \
+    crate://crates.io/zvariant_derive/3.10.0 \
+"
+
+do_install:append () {
```

Comment:
Move above `SRC_URI` (containing the crates) so it does not get overlooked?

Reply:
...ok